src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/MaterialsBroadcaster.cs

```diff
@@ -69,13 +71,20 @@ private MaterialPropertyAsset[] GetCachedMaterialProperties(int materialIndex, M
             return cachedMaterialPropertyAccessors[materialIndex];
         }
 
+        private bool ShouldAssessAnyPropertyAccessors(StateSynchronizationPerformanceParameters performanceParameters)
+        {
+            return !(!performanceParameters.HasCustomMateriaPropertyPollingFrequencies &&
+                performanceParameters.ShaderKeywords == StateSynchronizationPerformanceParameters.PollingFrequency.UpdateOnceOnStart &&
+                performanceParameters.MaterialProperties == StateSynchronizationPerformanceParameters.PollingFrequency.UpdateOnceOnStart);

```

Comment:
I think I'd find this easier to parse if the first ! was expanded over the rest of the conditions, e.g. "performanceParameters.HasCustomMateriaPropertyPollingFrequencies || performanceParameters.ShaderKeywords != StateSynchronizationPerformanceParameters.PollingFrequency.UpdateOnceOnStart || performanceParameters.MaterialProperties != StateSynchronizationPerformanceParameters.PollingFrequency.UpdateOnceOnStart